src/components/Button/index.js

```diff
@@ -74,11 +78,12 @@ const SbButton = {
   render(h) {
     const useTooltip = this.hasIconOnly && this.iconDescription
 
-    const renderIcon = (icon) => {
+    const renderIcon = (icon, color) => {
       return h(SbIcon, {
         props: {
           size: this.iconSize,
           name: icon,
+          color: color,
```

Comment:
You may use _object property value short-hand_ here

```suggestion
          color,
```